docs/_quarto.yml

```diff
@@ -428,7 +428,26 @@ quartodoc:
                 - strftime
                 - time
                 - truncate
-            - IntervalValue
+            - name: IntervalValue
+              dynamic: true
+              members:
+                - to_unit
+                - negate
+                - __add__
+                - __sub__
+                - __mul__
+                - __floordiv__
```

Comment:
```suggestion
```

We're probably going to get rid of these methods soon, they're extremely finicky and don't really work as advertised.